### ISSUE TITLE
client.sendMessage This method is not available

### ISSUE THREAD

**Issue by a20000110** [pedroslopez/whatsapp-web.js#2063]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

When I run this method, a successful result is returned, but the new message is not updated in time in my interface view, and I cannot use the fetchMessage method to get the newly sent message. I guess the sendMessage method does not send the new message

### Expected behavior

There should be new messages sent out, but I didn't see or get them

### Steps to Reproduce the Bug or Issue

Trigger sendMessage after successful login

### Relevant Code

 const content= new MessageMedia(
                    file.type,
                    await fileToBase64(file, false),
                    file.name,
                    file.size
                )
                const options = {caption: content}
                if (params.quotedMessageId) {
                    options.quotedMessageId = params.quotedMessageId
                }
await client.sendMessage(toAccount, content, options);

### Browser Type

Google Chrome

### WhatsApp Account Type

Standard

### Does your WhatsApp account have multidevice enabled?

No, I am not using Multi Device

### Environment

Windows

**Comment by a20000110** [comment]
> 请试试这个： [matricce@ 076f20a](https://github.com/matricce/whatsapp-web.js/commit/076f20a5d9aac21160f4abc57766dfe251841d19)

Thank you for your reply. In my project, I tried to modify the code you provided, but it did not solve the problem of unresponsive sending of media files.

**Comment by a20000110** [comment]
> 请试试这个： [matricce@ 076f20a](https://github.com/matricce/whatsapp-web.js/commit/076f20a5d9aac21160f4abc57766dfe251841d19)

I tried to call the sendMessage method to pass different parameters, so I can use the send file.
`
await WWebJS.sendMessage("123456789@c.us","hi",{
    attachment:{
     mimetype:'application/pdf',
    data:pdf,
    filename:'1.pdf'
    },
     caption: "hi",
})
`
However, I found a problem. When sending a pdf, the caption parameter is carried, and the value of this parameter cannot be displayed in the called whatsapp view. It seems that all types except pdf are normal.
